README.md

```diff
@@ -716,7 +716,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
 | [`Back To Top`](#contents) |
 
 - ### **D**
-
+  
```

Comment:
please remove the spaces here

Reply:
Alright @schmelto,  I have corrected it .

Reply:
Looks like the issue's still present. Do you need any help? 🙂